src/fetchers/top-languages-fetcher.js

```diff
@@ -37,14 +43,28 @@ const fetcher = (variables, token) => {
 async function fetchTopLanguages(username, exclude_repo = []) {
   if (!username) throw Error("Invalid username");
 
-  const res = await retryer(fetcher, { login: username });
+  const pageSize = 100;
```

Comment:
Maybe we can use the [pageInfo](https://docs.github.com/en/graphql/reference/objects#pageinfo) object to get cleaner code. 🚀 

```js
  // Loop through all pages and fetch all repos
  let repoNodes = [];
  let hasNextPage = true
  let endCursor = null
  while (hasNextPage) {
      const variables = { login: username, first: 100, after: endCursor };
      const res = await retryer(fetcher, variables);
      
      if (res.data.errors) {
        logger.error(res.data.errors);
        throw Error(res.data.errors[0].message || "Could not fetch user");
      }
      
      repoNodes.push(...res.data.data.user.repositories.nodes);
      hasNextPage = res.data.data.user.repositories.pageInfo.hasNextPage
      endCursor = res.data.data.user.repositories.pageInfo.endCursor
  }
```